docs/workflows/common-workflow-steps/sauce-labs-saucectl.md

```diff
@@ -22,25 +22,15 @@ Below are the workflow steps required before running the **Saucectl** step, list
 | --------------------------- | ----------------------------------------------------------------------------- |
 | [Android Build for UI Testing](/workflows/android-specific-workflow-steps/android-build-for-ui-testing) | This step generates the required Android test application outputs needed for testing on Sauce Labs. |
 
-<Screenshot url='https://cdn.appcircle.io/docs/assets/android-workflow-components-saucectl-1.0.png'/>
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE3930-androidSauce1.png' />
```

Comment:
Screenshot is broken here.
![Screenshot 2024-09-11 at 11 48 42](https://github.com/user-attachments/assets/ea042cd9-6d6a-42d4-9dcf-10c7597c0eef)